README.md

```diff
@@ -1,2 +1,26 @@
-# tsdbcg
-A Deck Building Card Game built using X and Y working with A to Z.
+# Endpoints
+## Get `api/v1/gamestate/:game_id`
+
+Successfully returns a list of all cards as an array of objects. Empty values are returned as nil. in the format
+```
+{
+    name: "Copper",
+    category: "Money",
+    cost: 0,
+    victoryPoints: nil,
+    spendingPower: 1,
+    numberAvailable: 47
+    id: 1113
+  },
+  {
+    name: "Estate",
+    category: ["Victory"],
```

Comment:
For the time being, can we get the brackets off of this category? I don't think we'll be implementing multi-category Cards until later down the road.

Reply:
Checking in with the Frontend team, turns out that I was wrong. Refer to #28 for more information.